apps/test-viewer/src/components/Viewer.tsx

```diff
@@ -66,29 +66,28 @@ function useViewCreatorOptions() {
    */
   const viewConfiguration = useCallback((viewPort: ScreenViewport) => {
     // default execute the fitview tool and use the iso standard view after tile trees are loaded
-    const tileTreesLoaded = () => {
-      return new Promise((resolve, reject) => {
-        const start = new Date();
-        const intvl = setInterval(() => {
-          if (viewPort.areAllTileTreesLoaded) {
-            ViewerPerformance.addMark("TilesLoaded");
-            ViewerPerformance.addMeasure("TileTreesLoaded", "ViewerStarting", "TilesLoaded");
-            clearInterval(intvl);
-            resolve(true);
-          }
-          const now = new Date();
-          // after 20 seconds, stop waiting and fit the view
-          if (now.getTime() - start.getTime() > 20000) {
-            reject();
-          }
-        }, 100);
-      });
-    };
-
-    tileTreesLoaded().finally(() => {
-      void IModelApp.tools.run(FitViewTool.toolId, viewPort, true, false);
-      viewPort.view.setStandardRotation(StandardViewId.Iso);
-    });
+    // const tileTreesLoaded = () => {
```

Comment:
Is this intentionally commented out and could it be deleted if it's unused?

Reply:
Actually this is causing rejected promise error in the application because it times out.
@aruniverse Do we have any better way to achieve this?

Reply:
i would just remove this, its not really needed. just added as an example in the starter templates

Reply:
@aruniverse @eringram ok removed `useViewCreatorOptions`

Reply:
I think the Viewer itself loads this view config if nothing is passed in. It has been a while since I've checked but I think it still holds true. https://github.com/iTwin/viewer/blob/a58000781f538ac468368e875072d587a415186d/packages/modules/viewer-react/src/services/iModel/ViewCreator3d.ts#L71

When I had to deal with this, I would just pass in an empty object into `viewportConfigurer`.